### ISSUE TITLE
We should add a keyboard functional skip link to skip the navigation

### ISSUE THREAD

**Issue by SaptakS** [freedomofpress/securedrop.org#872]
We should add a keyboard functional skip link to skip the navigation and jump directly to the main content. Few things to keep in mind:

- The skip link should be selectable by keyboard, screen readers, etc.
- The skip link should be the first focusable element on the page
- It should be the first element inside the <body> tag
- Preferably always be visible, but at least should be visible on keyboard focus.

WCAG [2.4.1](https://www.w3.org/TR/UNDERSTANDING-WCAG20/navigation-mechanisms-skip.html)
